src/scripts/eleventy-patterns/navigation.js

```diff
@@ -0,0 +1,41 @@
+import headsUp from 'headsup.js';
+
+// Toggle menu on small screens
+function initMenuToggle() {
+  const menuToggle = document.querySelector('.EP_nav__menu-toggle');
+  const menu = document.querySelector('.EP_nav__menu');
+
+  menuToggle.addEventListener('click', () => {
+    let expanded = menuToggle.getAttribute('aria-expanded') === 'true' || false;
+    menu.classList.toggle('is-open');
+    menuToggle.setAttribute('aria-expanded', !expanded);
+    document.body.classList.toggle('no-scroll');
+  });
+}
+
+// Toggle menu accordions
+function initSubmenuToggle() {
+  const submenuToggle = document.querySelectorAll('.EP_nav__submenu-toggle');
+
+  submenuToggle.forEach(toggle => {
+    toggle.addEventListener('click', event => {
+      event.preventDefault();
+      let target = toggle.nextElementSibling;
+      let expanded = toggle.getAttribute('aria-expanded') === 'true' || false;
+      const toggleIcon = toggle.querySelector('.toggle-icon');
+      target.classList.toggle('is-open');
+      toggleIcon.classList.toggle('is-open');
+      toggle.setAttribute('aria-expanded', !expanded);
+    });
+  });
+}
+
+window.addEventListener('load', () => {
+  initMenuToggle();
+  initSubmenuToggle();
+  headsUp({
+    selector: '.EP_header-container',
+    debounce: 100,
+    hiddenHeaderClass: 'EP-nav-hidden'
+  });
+});
```

Comment:
I wonder if this file should be imported into the main pattern lib JS bundle (I haven't viewed all the files yet, so maybe it is!)

Reply:
I think this is probably fine as-is for now. If we add more pattern lib JS we may rethink this.